test/test_kaldi_dirs.py

```diff
@@ -0,0 +1,305 @@
+
+
+@pytest.fixture
+def multi_channel_recording():
+    recording = lhotse.RecordingSet.from_recordings(
+        [
+            lhotse.Recording(
+                id="lbi-1272-135031-0000",
+                sources=[
+                    lhotse.AudioSource(
+                        type="file", channels=[0, 1], source="nonexistent.wav"
+                    ),
+                ],
+                sampling_rate=16000,
+                num_samples=174160,
+                duration=174160.0 / 16000.0,
+                transforms=None,
+            )
+        ]
+    )
+
+    supervision = lhotse.SupervisionSet.from_segments(
+        [
+            lhotse.SupervisionSegment(
+                id="lbi-1272-135031-0000-A",
+                recording_id="lbi-1272-135031-0000",
+                start=0.0,
+                duration=10.885,
+                channel=0,
+                text="SOMETHING",
+                speaker="lbi-1272-135031-A",
+                gender="m",
+            ),
+            lhotse.SupervisionSegment(
+                id="lbi-1272-135031-0000-B",
+                recording_id="lbi-1272-135031-0000",
+                start=0.0,
+                duration=10.885,
+                channel=1,
+                text="SOMETHING ELSE",
+                speaker="lbi-1272-135031-B",
+                gender="f",
+            ),
+        ]
+    )
+    return recording, supervision
+
+
+@pytest.mark.xfail(reason="multi fail recordings not supported yet")
+def test_multi_file_recording(tmp_path, multi_file_recording):
+    with working_directory(tmp_path):
+        lhotse.kaldi.export_to_kaldi(
+            multi_file_recording[0], multi_file_recording[1], ".", None, False
+        )
+
+
+def test_multi_channel_recording(tmp_path, multi_channel_recording):
+    with working_directory(tmp_path):
+        lhotse.kaldi.export_to_kaldi(
+            multi_channel_recording[0], multi_channel_recording[1], ".", None, False
+        )
+
+
+@contextlib.contextmanager
+def working_directory(path):
+    """Changes working directory and returns to previous on exit."""
+    prev_cwd = Path.cwd()
+    os.chdir(path)
+    try:
+        yield
+    finally:
+        os.chdir(prev_cwd)
+
+
+@pytest.mark.parametrize("replace", [None, "b"])
+def test_kaldi_import(replace):
+    fixture_path = MINILIB_PATH
+    with working_directory(fixture_path):
+        out = lhotse.kaldi.load_kaldi_data_dir(fixture_path, 16000, 0.01, replace, 1)
+
+    lhotse_dir = "lhotse"
+    if replace:
+        lhotse_dir += "-" + replace
+
+    with working_directory(fixture_path / lhotse_dir):
+        recording_set = lhotse.RecordingSet.from_jsonl("recordings.jsonl.gz")
+        supervision_set = lhotse.SupervisionSet.from_jsonl("supervisions.jsonl.gz")
+
+    assert out[0] == recording_set
+    assert out[1] == supervision_set
+
+
+@pytest.mark.xfail(reason="Import ordering is somehow not working for now")
+def test_get_duration_failurex(monkeypatch):
```

Comment:
will fix imports (for future self: `from augmentation import *` is broken in top-level `__init__.py`)